chevah_build

```diff
@@ -268,12 +314,12 @@ case $OS in
         export BUILD_LIBEDIT="no"
         ;;
     rhel5|sles11)
-        # Up-to-date cryptography not working on these...
+        # cryptography 1.3.4 is latest version with support for OpenSSL 0.9.8.
         PIP_LIBRARIES="\
             setproctitle==1.1.10 \
-            cryptography==1.3.1 \
+            cryptography==1.3.4 \
             pyOpenSSL==16.0.0 \
```

Comment:
Can we use `pyOpenSSL==17.0.0` here? Maybe we should add a comment why it can't be used

Reply:
I have another ticket for that: https://trac.chevah.com/ticket/4838. It's actually the reason for updating cryptography in the first place, as it's a security bug.

Reply:
But actually yes, in this particular case, I think pyOpenSSL version 16.0.0 is the latest version with support for cryptography 1.3.4. More work on documenting these will come in the aforementioned ticket.